index.html

```diff
@@ -4928,13 +4928,21 @@ <h3>Fitness for Purpose</h3>
     <section class="appendix informative">
       <h2>Contexts, Types, and Credential Schemas</h2>
 
-      <section class="informative">
+      <section class="normative">
         <h3>Base Context</h3>
+
+        <p class="issue" title="(AT RISK) Hash values might change during Candidate Recommendation">
+This section lists cryptographic hash values that might change during the
+Candidate Recommendation phase based on implementer feedback that requires
+the referenced files to be modified.
+        </p>
           <p>
-The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
-with a SHA-256 digest of
-<strong><code>ff1fdfa8d3f07547ec149b6652fe3f5a4276b9f0c48344dc29400763ba7b44fa</code></strong>,
-can be used to implement a local cached copy. It is possible to confirm the
+Implementations MUST ensure that the base context value, located at
+<code>https://www.w3.org/ns/credentials/v2</code>, matches the following
+SHA-256 digest of the value:
```

Comment:
What are implementations supposed to do if they don't match?

Reply:
For the base context... almost certainly throw an error (if you are not using a cached copy, which all VC processors should be using).

@jyasskin, we did have approval to merge this on a call today, would you prefer that we don't and address this particular concern?

Reply:
I don't have any opinion about the timing of fixing this (i.e. feel free to merge and just queue the issue), but I think the spec will need to eventually tell the processors what to do instead of just saying they need to check. This might need a change to 5.2.5 in https://www.w3.org/TR/json-ld-api/#algorithm to make it possible to check contexts against a hash value when fetching them, or you could possibly require that all implementations treat this URL as "previously dereferenced" (step 5.2.4) with a particular value, in which case you don't need the hash at all.

Reply:
If the URL is previously dereferenced, it doesn't need to be a URL, it could be a URN.

Context needs to dereference to the same bytes... Context identifier does not actually matter... 

Using a URN, or some other IRI would also protect W3C from being forced to serve the JSON-LD file that is always assumed to be previously dereferenced over and over again.

Making the value a URL encourages traffic, and implies it will be resolvable forever... It's not just a security thing, there is a cost / sustainability component to how this is made normative as well.

Reply:
@OR13,

> If the URL is previously dereferenced, it doesn't need to be a URL, it could be a URN.

We've had this discussion before (setting aside also that a "URN" is now also a "URL" per WHATWG). To increase interoperability in the linked data ecosystem, it should be a URL. Only the people who come, read, and implement the VCDM spec will be able to (and should be required to, IMO) treat the context as already dereferenced. We should not block other processors written by people who haven't read the spec from being able to consume VC data as general linked data.